ex7/ex7.js

```diff
@@ -9,3 +9,70 @@
  * mostFrequentChar("a b c d e f") === " "
  * mostFrequentChar("") === null
  */
+
+function mostFrequentChar(str) {
+
+	objCharCount = {};
+
+	if (str === '') {
+		return null;
+	}
+
+	objCharCount = countChar(str);
+
+	return getFrequentChar(objCharCount);
+}
+
+function getFrequentChar(objCharCount) {
+	
+	var maxCount = 0,
+	  frequentChar = null,
+	  frequentCount = 0,
+	  arrayChar = [],
+	  currentCount = 0,
+	  currentChar = '';
+
+	arrayChar = Object.keys(objCharCount);
+
+	for (var i = 0; i < arrayChar.length; i++) {
+
+		currentChar = arrayChar[i];
+		currentCount = objCharCount[currentChar];
+
+		if (maxCount < currentCount) {
+
+			maxCount = currentCount;
+			frequentChar = currentChar;
+
+		} else if (maxCount === currentCount) {
+
+			frequentChar = null;
```

Comment:
This is not clear. I would instead add another variable called `moreThanOneMax` and set it to true here (then check for it below).

Also, I think this will fail on this instance: 1122333.